kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnector.java

```diff
@@ -86,4 +89,21 @@ public ConfigDef config() {
     return S3SinkConnectorConfig.getConfig();
   }
 
+  @Override
+  public Config validate(final Map<String, String> connectorConfigs) {
+    Config config = super.validate(connectorConfigs);
+    // Checking whether the bucket exists
+    if (!S3BucketCheck.checkBucketExists(new S3SinkConnectorConfig(connectorConfigs))) {
+      for (ConfigValue configValue : config.configValues()) {
+        if (configValue.name().equals(S3SinkConnectorConfig.S3_BUCKET_CONFIG)) {
+          String bucketName = configValue.value().toString();
+          configValue.addErrorMessage(String.format("The S3 bucket named %s could not be found."
+                  + " Check the S3 credentials and verify the %s bucket exists.",
+              bucketName, bucketName));
```

Comment:
Nit: our style convention is to separate multi-parameter calls
```suggestion
          configValue.addErrorMessage(
              String.format(
                  "The S3 bucket named %s could not be found."
                  + " Check the S3 credentials and verify the %s bucket exists.",
                  bucketName,
                  bucketName
              )
          );
```